.config/forest.dic

```diff
@@ -17,3 +17,7 @@ tuple
 mutex
 RPC
 parameterize/D
+сonfig
+TODO
```

Comment:
`TODO` should never be part of the official Forest documentation. Move it to a dev comment or refactor it some other way.